src/plugins/app-route/handler.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import url from 'url';
+
+export default (languageRenderers: { [string]: LanguageRenderer$ }) => (
+	request: HapiRequest,
+	reply: HapiReply
+) => {
+	const pathname = request.getPrefixedPath();
```

Comment:
Makes sense - I'll do that in a separate branch and merge straight to `next` before this PR merges